automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/WorkspaceSelector.scala

```diff
@@ -2,7 +2,7 @@ package org.broadinstitute.dsde.firecloud.component
 
 import org.openqa.selenium.WebDriver
 
-case class WorkspaceSelector(implicit webDriver: WebDriver) extends Component(TestId("workspace-selector")) {
+case class WorkspaceSelector()(implicit webDriver: WebDriver) extends Component(TestId("workspace-selector")) {
```

Comment:
Compile error:
`[error] /app/src/test/scala/org/broadinstitute/dsde/firecloud/component/WorkspaceSelector.scala:5:29: case classes should have a non-implicit parameter list; adapting to 'case class WorkspaceSelector()(...)'`